.scalafmt.conf

```diff
@@ -1,4 +1,23 @@
 version=2.4.2
 
-align = true    # For pretty alignment.
+align = more    # For pretty alignment.
 maxColumn = 140 # For my wide 30" display.
+
+rewrite.rules = [
+  SortModifiers
```

Comment:
Uhm, not a fan of the default on this one (for some reason I find visibility last particularly hard to get used to), I'd be more like 90% in agreement with https://docs.scala-lang.org/style/declarations.html#modifiers - aside all annotations on their own line (I like my `@tailrec` to be where i put it, for example, so happy to have `@tailrec def loop[A]: A = ???`). 

I tend to like it more like
```
rewrite.sortModifiers.order = [
  "override", "private", "protected", "implicit", "final", "sealed", "abstract", "lazy"
]
```
but if you happen to prefer the default `implicit, final, sealed, abstract, override, private, protected, lazy` I'd like to have your perspective and I'm happy to discuss.

Reply:
I'm more than happy to meet your preferences. I just want it to be automatically enforced by the rewrite so we don't have to think about it again. I will try to configure it to make it as above. 👍